### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by theilman7**
Creating pull request for[ Footer #75](https://github.com/baltimorecounty/dotgov-components/issues/75). Will need to upload corresponding css in the components repo as well.

### CODE REVIEW

File: src/components/SiteFooter.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  return (
+    <footer className="dg_county-footer">
+      <div className="container">
+        <div className="row">
+          <div className="col-lg-6 col-md-12 col-sm-12">
+            <div className="dg_brand-footer">
+              <div className="dg_seal-container">
+                <CountySeal />
+              </div>
+              <h1>
```
Comment (martypowell) [inline src/components/SiteFooter.js]: I think this needs to a p. I'm checking with @sdurphy to see if this is identical to teh header, if so we might want to create a component for it.

File: src/components/SiteFooter.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      <img
+        className="img-fluid"
+        src="http://staging.baltimorecountymd.gov/sebin/t/o/homepage-baltcogo-logo-dark.png"
+        alt="BaltCoGo Logo"
+      />
+    </div>
+    <div className="dg_footer-section">
+      <p className="dg_download-blurb">Download now on:</p>
+      <ul className="dg_alt-list inline">
+        <li>
```
Comment (martypowell) [inline src/components/SiteFooter.js:49]: this could be moved to the data file, but not blocking.

File: src/components/SiteFooter.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        href="https://www.google.com/maps?q=400+Washington+Avenue,+Towson+Maryland+21204&amp;um=1&amp;ie=UTF-8&amp;sa=X&amp;ved=0ahUKEwiOn7vd5qLgAhWpnOAKHWVfBqIQ_AUIDigB"
+      >
+        <i className="fa fa-map-marker-alt dg_lg_gray" aria-hidden="true"></i>
+        400 Washington Avenue, Towson Maryland 21204
+      </a>
+    </div>
+  </div>
+);
+
+const BaltcoGoSection = () => (
```
Comment (martypowell) [inline src/components/SiteFooter.js:37]: these could probably be moved to their own components, but not blocking.

**Review by martypowell — DISMISSED** [review]

**Comment by martypowell** [comment]
hey @sdurphy this is ready for a look at https://baltimorecounty.github.io/dotgov-prototype-2.0/. @theilman7 and I guessed at the responsiveness, so please provide feedback.

**Comment by sdurphy** [comment]
The responsiveness looks correct... is there any way to add more space between the h3's and the lists above them? (For example, if we could add more space above "policies")

**Comment by martypowell** [comment]
> The responsiveness looks correct... is there any way to add more space between the h3's and the lists above them? (For example, if we could add more space above "policies")

@sdurphy there is 30px margin below the lists, is that too little?

**Review by martypowell — APPROVED** [review]

**Pull request merged**